plugin/src/main/kotlin/com/exactpro/th2/gradle/ComponentTh2Plugin.kt

```diff
@@ -59,6 +60,10 @@ class ComponentTh2Plugin : Plugin<Project> {
                 if (version.toString().let { it == Project.DEFAULT_VERSION || it.isEmpty() }) {
                     throw GradleException("project '$name' missing version (use version property to provide the version)")
                 }
+                val mainClass = the<JavaApplication>().mainClass
+                if (!mainClass.isPresent || mainClass.get().isBlank()) {
```

Comment:
there is `getOrElse` method that can be used here to simplify the condition
```suggestion
                if (the<JavaApplication>().mainClass.getOrElse("").isBlank()) {
```